packages/optimizer/lib/transformers/AddBlurryImagePlaceholders.js

```diff
@@ -166,14 +165,16 @@ class AddBlurryImagePlaceholders {
           return image.getBase64Async('image/png');
         })
         .then((dataURI) => {
+          log.debug(img.attribs.src);
```

Comment:
nit: this is going to print an URL. Maybe it's worth adding more context on what it is?

Reply:
it's going to print: `AMP Optimizer AddBlurryImagePlaceholders bball.jpg`. Should I add more?

Reply:
Maybe the URL for the original image (if it's not being logged at some point before this)

Reply:
@andreban tweaked it a bit in the latest commit.

Reply:
Success case:

```
AMP Optimizer AddBlurryImagePlaceholders bball.jpg [SUCCESS]
```

Error case:

```
AMP Optimizer AddBlurryImagePlaceholders ERROR ENOENT: no such file or directory, open '/Users/sbenz/projects/amp-toolbox/packages/optimizer/spec/assets/bball2.jpg'

AMP Optimizer AddBlurryImagePlaceholders bball2.jpg [FAIL]
```